vaderSentiment/vaderSentiment.py

```diff
@@ -204,7 +204,7 @@ def make_lex_dict(self):
         Convert lexicon file to a dictionary
         """
         lex_dict = {}
-        for line in self.lexicon_full_filepath.split('\n'):
+        for line in self.lexicon_full_filepath.rstrip('\n').split('\n'):
```

Comment:
This is the only line that actually changed. The rest is my editor removing extra whitespaces. Sorry about that.